io/src/obj_io.cpp

```diff
@@ -197,11 +199,7 @@ pcl::MTLReader::read (const std::string& mtl_file_path)
         continue;
 
       // Tokenize the line
-      std::stringstream sstream (line);
-      sstream.imbue (std::locale::classic ());
-      line = sstream.str ();
```

Comment:
Is this noop? I didn't understand it. We create stringstream, copy line to underlying buffer, set locale and then copy underlying buffer back to the line. As I understand it, we are not changing line here in anyway and I think I confirmed it by checking out libcxx source code.

Reply:
I will also check this again, but yes, seems like this has no effect